defects.egg-info/PKG-INFO

```diff
@@ -0,0 +1,10 @@
+Metadata-Version: 1.0
+Name: defects
+Version: 0.1.0
+Summary: Introducing Missing-linker Defects in Metal-Organic Frameworks
+Home-page: https://github.com/meiirbek-islamov/defects
+Author: Meiirbek Islamov
+Author-email: mei12@pitt.edu
+License: UNKNOWN
+Description: UNKNOWN
+Platform: UNKNOWN
```

Comment:
.egg-info files should be in your .gitignore and shouldn't be committed to the repo.